homepage/templates/homepage/courses/courses.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+<title>Class Rater Courses</title>
+</head>
+
+<body>
+{% extends 'homepage/app_layout.html' %}
+{% block content %}
+<div class="container" style="margin-top:30px">
+  <div class="row">
+    <div class="col-sm-4">
+      <h2>Courses Page</h2>
+    </div>
+  </div>
+</div>
+{% endblock %}
+<p class="EOM">(End of messages)</p>

```

Comment:
Here as well, since it's going to be a table and the paragraph element is outside the block content